src/app/system/navigation/navigate.ts

```diff
@@ -153,7 +153,10 @@ export async function navigate(url: string, options: NavigateOptions = {}) {
     }
 
     if (waitForTabsToChange) {
-      requestAnimationFrame(pushView)
+      const timeoutTime = __TEST__ ? 0 : 200
+      setTimeout(() => {
+        requestAnimationFrame(pushView)
```

Comment:
we probably don't need this anymore since `requestAnimationFrame` translates to `setTimeout` with 60 ms` 

```suggestion
        pushView
```

Reply:
woops, just merged also from react native docs:

```md
requestAnimationFrame() for code that animates a view over time.
setImmediate/setTimeout() run code later, note this may delay animations.
runAfterInteractions() run code later, without delaying active animations.
```